### PR TITLE
Removes the Robot Console's ability to blow up borgs

### PR DESCRIPTION
**Pull request by Anewbe**
It's a weeeeee bit unethical to be able to explode a sapient thing on a whim, yeah? Plus it's probably cheaper to find and correct the borg than to remake/train a new one.

### CODE REVIEW

**Comment by PsiOmegaDelta** [comment]
Unless Polaris has changed it, it's technically only the chassis that's blown up.
The brain itself remains unharmed, if disoriented.

**Comment by Anewbe** [comment]
Still rude.

**Comment by EmperorJon** [comment]
Are there ever circumstances where you can blow a borg but not lock them down?

**Comment by Anewbe** [comment]
Not as far as I can tell, no. Traitor borgs were already supposed to be immune, this just extends that courtesy to regular borgs.

File: html/changelogs/anewbe.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscdel: "Removed joy"
```
Comment (PsiOmegaDelta) [inline html/changelogs/anewbe.yml]: This is a rather meaningless changelog.

Comment (Anewbe) [inline html/changelogs/anewbe.yml]: Oh hey, wrong changelog, one moment.

**Comment by EmperorJon** [comment]
I don't really see why we need to keep boom given that you can lock them down, and we specifically made it so you can't lock down traitors anyway because that's (rather correctly) deemed to be too easy a "permastun".
*shrug*

**Comment by Neerti** [comment]
Gets a :+1: from me.

**Pull request merged**